main_app/services.py

```diff
@@ -1,9 +1,15 @@
 from django.http import JsonResponse
 import requests
 
-def get_parks(self, pk):
-    info = requests.get('https://jsonplaceholder.typicode.com/todos/1').json()
+def get_parks(self, coordinates='', query=''):
```

Comment:
As of right now 'query' is set to whatever params the developer passes with the get request.